src/components/v2v3/V2V3Project/V2V3PayButton/V2V3ConfirmPayModal/V2V3ConfirmPayModal.tsx

```diff
@@ -262,7 +261,7 @@ export function V2V3ConfirmPayModal({
                     <Trans>
                       You receive these NFTs for contributing{' '}
                       <strong>
-                        {nftRewardTier.contributionFloor} ETH or more
+                        {sumTierFloors(nftRewardTiers)} ETH or more
```

Comment:
**suggestion** does it make sense to just remove `or more` now?

Reply:
I think it still makes sense. You can still pay more than each tier individually and also more than the sum of any, e.g: 
<img width="584" alt="Screen Shot 2022-11-20 at 12 04 25 pm" src="https://user-images.githubusercontent.com/96150256/202879238-5bff4152-3c04-462a-81b1-d462d6657346.png">

Reply:
sgtm